patches/net/minecraft/data/tags/TagsProvider.java.patch

```diff
@@ -64,7 +64,7 @@
 -                                    List<TagEntry> list1 = list.stream().filter(p_274771_ -> !p_274771_.verifyIfPresent(predicate, predicate1)).toList();
 +                                    List<TagEntry> list1 = java.util.stream.Stream.concat(list.stream(), tagbuilder.getRemoveEntries())
 +                                              .filter((p_274771_) -> !p_274771_.verifyIfPresent(predicate, predicate1))
-+                                              .filter(this::missing)
++                                              .filter(entry -> tagbuilder.getRemoveEntries().noneMatch(entry::equals) && this.missing(entry))
```

Comment:
...not sure how I managed to miss that myself now that you point it out. I was advised to simply adapt that line to make the extra check, but I realise this could have been avoided by simplt not using a concatenated stream to begin with.

Reply:
Yeah, uh, that's a good point, no clue how I missed that 😅